home-assistant: test freebox component

dotlambda opened the pull request:
<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

###### Motivation for this change


###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

dotlambda committed:
home-assistant: update component-packages.nix

dotlambda committed:
home-assistant: test freebox component

mweinelt commented:
>  home-assistant, home-assistant.passthru.tests on x86_64-darwin Queued — Waiting to run this check... 

Wonder if it would be sensible to set `meta.platform` to exclude darwin.

dotlambda commented:
> Wonder if it would be sensible to set `meta.platform` to exclude darwin.

Only because ofborg gets stuck? Doesn't Home Assistant run perfectly well on macOS?

mweinelt commented:
Lots of python bindings are not compatible with MacOS. The tl;dr for the home-assistant install on MacOS is: run this VM image. https://www.home-assistant.io/installation/macos/

dotlambda committed:
home-assistant: set platforms to linux

Upstream does not support running it on macOS:
https://www.home-assistant.io/installation/macos/
